integration-tests/reactive-db2-client/pom.xml

```diff
@@ -0,0 +1,265 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2020 IBM Corporation
```

Comment:
IBM and Red Hat both :)

Looks correct, but I believe we generally don't have such notices? I see the other reactive integration tests have them, but not other module.

Reply:
Yea I noticed the other reactive DB integration tests had this header, so I changed it to IBM since it was me who was creating the file

Reply:
I would be in favor of removing the header completely if possible.